build.sh

```diff
@@ -175,27 +186,66 @@ if [ "$MACHINE" == "ncar" ]; then
   fi
 fi
 
+# comma-separated list of files in src/framework that are needed to build $LININFRA (for FMS2, at least)
```

Comment:
Is there a reason for compiling only the subset of files under src/framework? As an alternative to the current implementation in this PR, could we compile all files under src/framework/, and hence, don't have to list individual files in build.sh?